__docs__/wonder-blocks-typography/heading.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react";
+import {Meta, StoryObj} from "@storybook/react";
+import {StyleSheet} from "aphrodite";
+import packageConfig from "../../packages/wonder-blocks-typography/package.json";
+
+import ComponentInfo from "../components/component-info";
+import TypographyArgTypes from "./typography.argtypes";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Heading} from "@khanacademy/wonder-blocks-typography";
+import {spacing} from "@khanacademy/wonder-blocks-tokens";
+
+/**
+New component for Thunderblocks headings.
+
+When wrapped in `<ThemeSwitcher theme="classroom">`, this component will use the
+Plus Jakarta Sans font family.
+
+The `size` prop will select a font size token based on our
+[REM font sizing scale](/?path=/docs/packages-tokens-typography--docs&globals=theme:thunderblocks#size).
+A corresponding line-height token will be automatically selected from our
+[line-height scale](/?path=/docs/packages-tokens-typography--docs&globals=theme:thunderblocks#lineHeight).
+
+The `weight` prop will match a font weight token based on the available
+[font weights](/?path=/docs/packages-tokens-typography--docs&globals=theme:thunderblocks#weight).
+Only `h1` through `h6` are allowed as tagNames for accessibility purposes.
+
+Other content should utilize `BodyText`, primarily in `p` tags.
+**/
+
+export default {
+    title: "Packages / Typography / Thunderblocks / Heading",
+    component: Heading,
+    parameters: {
+        componentSubtitle: (
+            <ComponentInfo
+                name={packageConfig.name}
+                version={packageConfig.version}
+            />
+        ),
+        chromatic: {
+            // Disabling because all typography components are covered together
+            // in the Typography stories
+            disableSnapshot: true,
+        },
+    },
+    argTypes: TypographyArgTypes,
+} as Meta<typeof Heading>;
+
+/**
+ * A dynamic example of the `Heading` component where you can select a size and
+ * weight via props. Defaults to `size="large"` and `weight="bold"`.
+ */
+export const Default: StoryObj<typeof Heading> = {
+    render: (args) => <Heading {...args} />,
+    globals: {theme: "thunderblocks"},
+    args: {
+        children: "Heading",
+        size: "large",
+        weight: "bold",
+    },
+};
+
+/**
+ * An example of the `Heading` component's `size` and `weight` prop combinations,
+ * mimicking the ones found in Figma Foundation specs.
+ */
+export const SizesAndWeights = {
```

Comment:
I find this story useful for visual regression tests since it has the different combinations! What do you think about enabling snapshots for this story instead of the [New Typography story](https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=682b904ba1d77d7bd773cdd6) which only shows one variant of Heading/Body.

Same thing for the `BodyText SizesAndWeights` story! 

It would also be useful to have visual regression tests for both OG and TB themes!

Reply:
I adjusted the story settings for the Typography components and opened an issue for the theme visual regression tests here: https://khanacademy.atlassian.net/browse/WB-1957

Reply:
For visual regression tests, you can use the IconButton example as reference. It allow us to create snapshots for both themes using the same story:

https://github.com/Khan/wonder-blocks/blob/main/__docs__/wonder-blocks-icon-button/icon-button-testing-snapshots.stories.tsx#L27-L33

Reply:
I added the modes to this story, and I see what you mean! That is going to be helpful.

Reply:
For some reason that I don't understand, I'm not able to see these new snapshots being captured in Chromatic.